### DISCUSSION TITLE
Suggestion: Date Range Picker

### DISCUSSION THREAD

**Discussion by omarluq** [ruby-ui/ruby_ui#143] [Ideas]
Love what you folks are doing with the UI lib much needed in the rails ecosystem.
A small suggestion, extending the date picker to support date ranges maybe something similar to https://react-tailwindcss-datepicker.vercel.app/demo would be an awesome addition.

Thank you much ❤️

**Comment by cirdes** [comment]
@omarluq, it would be nice to have data range support in the date picker, just like ShadCN does [here](https://ui.shadcn.com/docs/components/date-picker#date-range-picker).

Unfortunately, we don’t have anyone looking into that right now. If you’d like, you can try to add this feature.
